container/README.md

```diff
@@ -0,0 +1,82 @@
+# ABRT Analytics in Podman container
+
+**Easiest way how to run ABRT Analytics (known historically as FAF)**
+---
+
+## How to deploy
+
+Prerequisites:
+A postgres database with semver extension is needed. Using abrt/postgres-semver
+image is recommended. Persistent storage is provided by means of a podman volume.
+A Redis container is required to run and schedule faf actions from the web UI (see below).
+Communication between ABRT Analytics, the database and Redis is ensured by running all three containers
+in the same podman pod.
+`podman volume create <volume_name>`
+`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name <pod_name>`
+`podman run --pod <pod_name> -v /var/tmp/data:/var/lib/pgsql/data -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit postgres-semver`
```

Comment:
1) The formating is weird. Markdown puts it all on one line - two spaces make a new line.
2) The mount of dir doesn't work, volume should be used instead. 

```suggestion
`podman volume create <volume_name>`  
`podman pod create -p 5432:5432 -p 6379:6379 -p 8080:8080 --name <pod_name>`  
`podman run --pod <pod_name> -v <volume_name> -e POSTGRESQL_ADMIN_PASSWORD=scrt --name db -dit postgres-semver`
```

Reply:
Fixed.